src/DotNetWorker.Grpc/GrpcWorker.cs

```diff
@@ -284,5 +259,54 @@ internal static FunctionLoadResponse FunctionLoadRequestHandler(FunctionLoadRequ
 
             return response;
         }
+
+        internal static FunctionEnvironmentReloadResponse EnvironmentReloadRequestHandler(WorkerOptions workerOptions)
+        {
+            var envReloadResponse = new FunctionEnvironmentReloadResponse
+            {
+                Result = new StatusResult { Status = StatusResult.Types.Status.Success },
+                WorkerMetadata = GetWorkerMetadata()
+            };
+            envReloadResponse.Capabilities.Add(GetWorkerCapabilities(workerOptions));
+
+            return envReloadResponse;
+        }
+
+        private static WorkerMetadata GetWorkerMetadata()
```

Comment:
content of this method is existing code. Moved to this method so that it can be called from worker init handler and env reload response handler.